scilpy/utils/util.py

```diff
@@ -1,9 +1,47 @@
 # -*- coding: utf-8 -*-
 
+from dipy.io.utils import get_reference_info
 import numpy as np
 from numpy.lib.index_tricks import r_ as row
 
 
+def compute_distance_barycenters(ref_1, ref_2, ref_2_transfo=None):
+    """
+    Compare the barycenter (center of volume) of two reference object.
+    If a transformation is provided it will tranform the reference #2 and
+    return the distance before transformation and after transformation.
+
+    Parameters
+    ----------
+    ref_1: reference object
+        Any type supported by the sft as reference (e.g .nii of .trk).
+    ref_2: reference object
+        Any type supported by the sft as reference (e.g .nii of .trk).
+    ref_2_transfo: np.ndarray
+        Transformation that modifies the barycenter of ref_2.
+    Returns
+    -------
+    distance: float or tuple (2,)
```

Comment:
I changed it, now it always returns the tuple. If a user wants to simply know the distance between both image barycenter it could be done with np.eye(4) and taking any of the two values of the tuple.